frontend/libs/studio-components/src/components/StudioProfileMenu/components/StudioProfileMenuButton/StudioProfileMenuButton.tsx

```diff
@@ -1,24 +1,19 @@
 import React, { type ReactElement } from 'react';
-import { DropdownMenu } from '@digdir/designsystemet-react';
+import { DropdownMenu, type DropdownMenuItemProps } from '@digdir/designsystemet-react';
 import classes from './StudioProfileMenuButton.module.css';
 
 export type StudioProfileMenuButtonProps = {
   itemName: string;
   isActive?: boolean;
-  onClick: () => void;
-};
+} & DropdownMenuItemProps;
 
 export const StudioProfileMenuButton = ({
   itemName,
   isActive,
-  onClick,
+  ...rest
 }: StudioProfileMenuButtonProps): ReactElement => {
   return (
-    <DropdownMenu.Item
-      key={itemName}
-      onClick={onClick}
-      className={isActive ? classes.selected : undefined}
-    >
+    <DropdownMenu.Item key={itemName} className={isActive ? classes.selected : undefined} {...rest}>
```

Comment:
`role='menuitemradio'` kan vel plasseres her, siden den er lik for alle? Siden vi brukler denne rollen, må vi også følge [reglene for `menuitemradio`](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/menuitemradio_role). Det innebærer å legge til `aria-checked={isActive}`. Da kan vi også bruke `toBeChecked` i testene i stedet for å sjekke mot klassen.